app/helpers/settings_helper.rb

```diff
@@ -1,39 +1,28 @@
 module SettingsHelper
-  def short_cat(category)
-    category.gsub(/Setting::/, '')
-  end
-
-  def cat_label(category)
-    category.constantize.humanized_category || short_cat(category)
-  end
-
   def grouped_settings(settings)
     settings.each_with_object({}) do |setting, memo|
```

Comment:
please implement https://github.com/theforeman/foreman/pull/8002#discussion_r618333544

Reply:
Note, this doesn't need the compact anymore as we are operating over the setting from registry.